sdl_viewer/src/main.rs

```diff
@@ -131,6 +140,10 @@ struct NodeView {
 
 impl NodeView {
     fn new(program: &GlProgram, node_data: octree::NodeData) -> Self {
+        unsafe{
```

Comment:
formatting: space before {.

Reply:
I stopped reviewing here as this still shows the diff of the IO thread change. This one should be merged soon, so once you rebase I'll have another look.